Layers/Layer.gd

```diff
@@ -25,6 +25,7 @@ enum RenderType {
 	PATH,
 	CONNECTED_OBJECT,
 	POLYGON,
+	GRADIENT_POLYGON,
```

Comment:
Is this really necessary? For the buildings it was not necessary to create a new enum type.